src/point.h

```diff
@@ -72,6 +72,12 @@ struct point {
         return point( x / rhs, y / rhs );
     }
 
+#ifndef CATA_NO_STL
+    inline point abs() const {
```

Comment:
`inline` is redundant; definitions inside the class definition are implicitly `inline`.

Reply:
Gah, the whole mandatory inline thing for non-members keeps tripping me up, will fix.